docs/guide/build.md

```diff
@@ -11,7 +11,7 @@ The production bundle assumes support for modern JavaScript. By default, Vite ta
 - Safari >=14
 - Edge >=88
 
-You can specify custom targets via the [`build.target` config option](/config/build-options.md#build-target), where the lowest target is `es2015`.
+You can specify custom targets via the [`build.target` config option](/docs/config/build-options.md#buildtarget), where the lowest target is `es2015`.
```

Comment:
It works fine for me:

https://main.vitejs.dev/guide/build.html#browser-compatibility

https://main.vitejs.dev/config/build-options.html#build-target

Reply:
Yes, in VitePress, `buildtarget` doesn't work. Maybe you are thinking about fixing the link when viewing this markdown in GitHub? https://github.com/maneesht/vite/blob/main/docs/guide/build.md

It is an interesting idea if that is the case, but it should be done by a GitHub feature that lets us configure how these paths work.

Reply:
@patak-dev - yes, that is correct. I was visiting the docs from GitHub itself.